src/components/images/UploadImage.js

```diff
@@ -1,21 +1,27 @@
-//Unable to make it happen with Redux - might try to refactor to Redux way later
+//This is still not a Redux form - will try to refactor to Redux way later
 
 import React, { Component } from "react";
-import LocalApi from "./../../apis/LocalApi";
+import LocalApi from "../../apis/LocalApi";
 import { Form, Button } from "react-bootstrap";
+// import 'fetchImages' here only to make auto refresh happens (in this way, we don't need to un-comment 'event.preventDefault()' on 'fileUploadHandler' for better user experience)

```

Comment:
See comment in code. I use 'fetchImages' so that we can still use 'event.preventDefault()'. When the page refreshes, it's less notice able to users.